CONTRIBUTING.md

```diff
@@ -103,17 +114,22 @@ Pull requests will be evaluated against a seven point checklist:
     can get with `install_github("klutometis/roxygen")`. This will be
     available on CRAN in the near future.
 
-<!--
 1.  If fixing a bug or adding a new feature to a non-graphical function,
-    please add a [testthat](https://github.com/hadley/testthat) unit test.
+    please add a [testthat](https://github.com/r-lib/testthat) unit test.
 
 1.  If fixing a bug in the visual output, please add a visual test.
     (Instructions to follow soon)
--->
 
 1.  If you're adding a new graphical feature, please add a short example
     to the appropriate function.
 
-This seems like a lot of work but don't worry if your pull request isn't perfect. It's a learning process and Winston and I will be on hand to help you out. A pull request is a process, and unless you've submitted a few in the past it's unlikely that your pull request will be accepted as is.
+This seems like a lot of work but don't worry if your pull request isn't perfect.
+It's a learning process and Winston and I will be on hand to help you out. 
```

Comment:
Maybe replace "Winston and I" by "members of the ggplot2 team"?

Reply:
good call. will do.